internal/provider/trigger_resource_test.go

```diff
@@ -405,49 +404,6 @@ resource "honeycombio_trigger" "test" {
 	})
 }
 
-func TestAcc_TriggerResourceThresholdFloatingPointBug(t *testing.T) {
```

Comment:
Rather than keeping this as a dedicated test I just rolled the important bit (the floating point math) into the test above it.